src/Tribe/RSVP.php

```diff
@@ -1359,7 +1365,11 @@ public function send_tickets_email( $order_id, $event_id = null ) {
 		 */
 		$content = apply_filters( 'tribe_rsvp_email_content', $this->generate_tickets_email_content( $to_send ), $event_id, $order_id );
 
-		wp_mail( $to, $subject, $content, $headers, $attachments );
+		$sent = wp_mail( $to, $subject, $content, $headers, $attachments );
+
+		if ( $sent ) {
+			$this->update_ticket_sent_counter( $all_attendees[0]['qr_ticket_id'], $this->attendee_ticket_sent );
```

Comment:
This should loop through `$all_attendees` array and update for each ticket ID in that list